### PR TITLE
Add new route for organisational request overview

### PR DESCRIPTION
**Pull request by rnugraha**
- Provide different route for organisational request overview
- Initialize unit testing for request-overview

### CODE REVIEW

File: podium-gateway/src/main/webapp/app/request/overview/request-overview.route.ts [hunk 1]
```diff
@@ -36,9 +36,26 @@ export const requestOverviewRoute: Route = {
...
+    path: 'organisation-overview',
+    component: RequestOverviewComponent,
+    resolve: {
+        'pagingParams': RequestResolvePagingParams
+    },
+    data: {
+        authorities: ['ROLE_ORGANISATION_COORDINATOR'],
+        pageTitle: 'request.pageTitle',
+        pageHeader: 'request.overview.pageHeaderOverview',
+        breadcrumb: 'organisation-overview',
```
Comment (berndvdveen) [inline podium-gateway/src/main/webapp/app/request/overview/request-overview.route.ts]: Please format as 'organisation overview'

File: podium-gateway/src/main/webapp/app/shared/request/request.service.ts [hunk 2]
```diff
@@ -12,6 +12,7 @@ import { Http, Response, URLSearchParams, BaseRequestOptions } from '@angular/ht
 import { Observable } from 'rxjs/Rx';
 import { RequestDetail } from './request-detail';
 import { RequestBase } from './request-base';
+import { Res } from 'awesome-typescript-loader/dist/checker/protocol';
```
Comment (berndvdveen) [inline podium-gateway/src/main/webapp/app/shared/request/request.service.ts]: Not used

**Comment by berndvdveen** [comment]
Its better to wait till the request overview for organisations is implemented with an actual call

**Comment by rnugraha** [comment]
There's build error

File: podium-gateway/src/main/java/nl/thehyve/podium/web/rest/RequestResource.java [hunk 3]
```diff
@@ -167,60 +167,107 @@
...
-        HttpHeaders headers = PaginationUtil.generatePaginationHttpHeaders(page, "/api/requests");
+        HttpHeaders headers = PaginationUtil.generatePaginationHttpHeaders(page, "/api/requests/requester");
         return new ResponseEntity<>(page.getContent(), headers, HttpStatus.OK);
     }
 
     /**
-     * Fetch the request
+     * GET  /requests/status/:status/requester : get all the requests for a requester with the status.
      *
-     * FIXME: UNSECURED - Add new annotation that check whether an member (coordinator or reviewer) of an organisation
```
Comment (berndvdveen) [inline podium-gateway/src/main/java/nl/thehyve/podium/web/rest/RequestResource.java]: @gijskant Wouldnt we want to secure this using the annotations as well? Even before calling the route. Maybe leave in the FIXME

File: podium-gateway/src/main/webapp/app/shared/request/request.service.ts [hunk 4]
```diff
@@ -41,9 +41,16 @@ export class RequestService {
         });
     }
 
-    findSubmittedRequests(req?: any): Observable<Response> {
+    findMySubmittedRequests(req?: any): Observable<Response> {
         let options = this.createRequestOption(req);
-        return this.http.get(`${this.resourceUrl}/status/Review`, options).map((res: Response) => {
+        return this.http.get(`${this.resourceUrl}/status/Review/requester`, options).map((res: Response) => {
```
Comment (berndvdveen) [inline podium-gateway/src/main/webapp/app/shared/request/request.service.ts]: @gijskant @rnugraha Submitted requests are not just the ones in Review. Maybe consider renaming

**Comment by berndvdveen** [comment]
Looks good to me

**Pull request merged**